ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -370,16 +373,28 @@ inline double getCacheMaxAgeDuration(std::string cacheControlData) {
 
 void RSkComponentImage::requestNetworkImageData(ImageSource source) {
   auto sharedCurlNetworking = CurlNetworking::sharedCurlNetworking();
+  networkRequestMutext.lock();
+  if(isRequestinProgress_ == true && remoteCurlRequest_!=nullptr ){
```

Comment:
JUst my Suggestion : it will be better to enclose the conditions in bracket for quick readability